### PR TITLE
fix(release-notes): updated release notes, screenshots

### PR DESCRIPTION
**Pull request by mcoker**

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -4,6 +4,35 @@ section: developer-resources
...
 ---
 
+## 2021.16 release notes (2021-12-07)
+Packages released:
+- [@patternfly/patternfly@v4.164.1](https://www.npmjs.com/package/@patternfly/patternfly/v/4.164.1)
+
+### Components
+- **Button:** Added demo for progress button completion state ([#4528](https://github.com/patternfly/patternfly/pull/4528))
+- **Card:** Replaced dropdowns with selects in demos ([#4520](https://github.com/patternfly/patternfly/pull/4520))
+- **Description list:** Added columnar layout ([#4544](https://github.com/patternfly/patternfly/pull/4544))
```
Comment (evwilkin) [inline RELEASE-NOTES.md:14]: columnar?

**Review by evwilkin — APPROVED** [review]

**Review by nicolethoen — APPROVED** [review]

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -4,6 +4,37 @@ section: developer-resources
...
+- **Dropdown:**
+  - Added plain text variant ([#4542](https://github.com/patternfly/patternfly/pull/4542))
+  - Added disabled styles ([#4543](https://github.com/patternfly/patternfly/pull/4543))
+- **Form:** Added support for horizontal layout at various breakpoints ([#4536](https://github.com/patternfly/patternfly/pull/4536))
+- **Multiplefileupload:** Added component ([#4548](https://github.com/patternfly/patternfly/pull/4548))
```
Comment (nicolethoen) [inline RELEASE-NOTES.md:19]: should this be Three words?

Comment (mcoker) [reply RELEASE-NOTES.md]: doh! updated.

**Review by nicolethoen — APPROVED** [review]

**Review by evwilkin — APPROVED** [review]
LGTM!

**Pull request merged**